config/samples/gateway-with-dataplane-metrics.yaml

```
@@ -0,0 +1,164 @@
+kind: GatewayConfiguration
+apiVersion: gateway-operator.konghq.com/v1beta1
+metadata:
+  name: kong
+  namespace: default
+spec:
+  dataPlaneOptions:
+    deployment:
+      replicas: 1
+      podTemplateSpec:
+        metadata:
+          labels:
+            dataplane-pod-label: example
+          annotations:
+            dataplane-pod-annotation: example
+        spec:
+          containers:
+          - name: proxy
+            # renovate: datasource=docker versioning=docker
+            image: kong/kong-gateway:3.10
```

We've recently migrated samples to use the OSS `kong` image in https://github.com/Kong/gateway-operator/pull/1569 so let's stick to that here as well:

```suggestion
            image: kong:3.9
```